### DISCUSSION TITLE
What is your favorite Ghibli Movie?

### DISCUSSION THREAD

**Discussion by takutoIMD** [NAIST-SE/AttractingContributors#185] [General]
I want to know which Ghibli Movie is most loved among us.
Please add the title to the list of my README.md and make a Pull Request!

https://github.com/takutoIMD/AttractingContributors-Ghibli_Movie/blob/main/README.md

**Comment by tktkbohshi** [comment]
Added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music

**Comment by TakumiHiraide** [comment]
Added!
Please help me to~
https://github.com/TakumiHiraide/My-favorite-movie

**Comment by pgDora56** [comment]
I have sent a PR, please check it!

I need your help! Tell me your favorite youtuber!
https://github.com/pgDora56/AttractingContributors-Youtuber

**Comment by TakumiHiraide** [comment]
Added!
Please help me to~
https://github.com/TakumiHiraide/My-favorite-movie

**Comment by veekay7** [comment]
Added!

I need your help, please tell me the country/countries you would love to visit in the future.
Cheers.

https://github.com/VantelinLuca/AttractingContributors-WishCountry
